src/core/sock/sockinfo.h

```diff
@@ -412,12 +412,6 @@ class sockinfo : public socket_fd_api,
     void remove_cqfd_from_sock_rx_epfd(ring *p_ring);
     int os_wait_sock_rx_epfd(epoll_event *ep_events, int maxevents);
     virtual bool try_un_offloading(); // un-offload the socket if possible
-    virtual inline void do_wakeup()
```

Comment:
so maybe after removing that, we can make it regular function instead of virtual.
we use it only in wakeup_pipe.